python/cusignal/convolution/convolve.py

```diff
@@ -219,7 +219,7 @@ def fftconvolve(in1, in2, mode="full", axes=None):
     >>> fig, (ax_orig, ax_mag) = plt.subplots(2, 1)
     >>> ax_orig.plot(cp.asnumpy(sig))
     >>> ax_orig.set_title('White noise')
-    >>> ax_mag.plot(np.arange(-len(sig)+1,len(sig)), autocorr)
+    >>> ax_mag.plot(cp.asnumpy(cp.arange(-len(sig)+1,len(sig))), cp.asnumpy(autocorr))
```

Comment:
We're getting a Python Style warning here. Do you mind changing this line from:

```python
ax_mag.plot(cp.asnumpy(cp.arange(-len(sig)+1,len(sig))), cp.asnumpy(autocorr))
```

to

```python
ax_mag.plot(cp.asnumpy(cp.arange(-len(sig)+1,len(sig))),
    cp.asnumpy(autocorr))
```

You can install flake8 into your conda environment and run:

`flake8 convolve.py` as well to ensure our coding style checks pass.